### PR TITLE
Feat(InvestigationTable): Modify columns

### PR DESCRIPTION
**Pull request by doronlnd84**
For  user stories:
1. 1158 - add SubOccupation (מוסד) column
2. 1128 - remove Priority column
3. 1149 - remove County column

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTablesHeaders.ts [hunk 1]
```diff
@@ -48,11 +51,12 @@ export const TableHeaders: IndexedInvestigation = {
     [TableHeadersNames.age]: 'גיל',
     [TableHeadersNames.city]: '	עיר מגורים',
     [TableHeadersNames.investigatorName]: 'חוקר מבצע',
-    [TableHeadersNames.county]: 'נפה מבצעת',
+    // [TableHeadersNames.county]: 'נפה מבצעת',
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTablesHeaders.ts:54]: why comment?

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 2]
```diff
@@ -302,7 +306,9 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
                                 const otherReason = user ? investigation?.investigationGroupReasonByGroupId?.otherReason : '';
                                 const subStatus = investigation.investigationSubStatusByInvestigationSubStatus ?
                                     investigation.investigationSubStatusByInvestigationSubStatus.displayName :
-                                    '';
+                                    ''; 
+                                const subOccupation=  investigation?.investigatedPatientByInvestigatedPatientId?.subOccupationBySubOccupation?.displayName;
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:310]: space before = and delete one space after like that:
`subOccupation = investigation?.investigatedPatientByInvestigatedPatientId...`

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 3]
```diff
@@ -302,7 +306,9 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
...
+                                const parentOccupation=  investigation?.investigatedPatientByInvestigatedPatientId?.subOccupationBySubOccupation?.parentOccupation;
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:311]: same

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTablesHeaders.ts [hunk 4]
```diff
@@ -3,6 +3,9 @@ import InvestigationMainStatus from 'models/InvestigationMainStatus';
 
 export const investigatorIdPropertyName = 'investigatorId';
 
+
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTablesHeaders.ts:6]: delete rows

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 5]
```diff
@@ -760,7 +707,7 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                                         onClick={(event: any) => handleCellClick(event, key, indexedRow.epidemiologyNumber)}
                                                     >
                                                         {
-                                                            getTableCell(key, indexedRow)
+                                                            getTableCell(key, indexedRow,index)
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:710]: space after ,

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 6]
```diff
@@ -784,7 +731,7 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                                                     onClick={(event: any) => handleCellClick(event, key, indexedGroupedInvestigationRow.epidemiologyNumber)}
                                                                 >
                                                                     {
-                                                                        getTableCell(key, indexedGroupedInvestigationRow)
+                                                                        getTableCell(key, indexedGroupedInvestigationRow,index)
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:734]: same

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 7]
```diff
@@ -397,26 +358,12 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
-                        {indexedRow.isComplex && <ComplexityIcon tooltipText={complexInvestigationMessage} />}
-                        <span className={cssClass}>
-                            {indexedRow[cellName as keyof typeof TableHeadersNames] || noPriorityMessage}
-                        </span>
-                    </div>
-                );
+              case TableHeadersNames.subOccupation:
+                  const subOccupation = indexedRow[cellName as keyof typeof TableHeadersNames];
+                  const parentOccupation = Boolean(subOccupation) ?  tableRows[index].parentOccupation : '';
+                  return    <Tooltip title={parentOccupation} placement="top">
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:364]: single quote 
`placement='top'`

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 8]
```diff
@@ -684,6 +639,9 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
                         const subStatus = investigation.investigationSubStatusByInvestigationSubStatus ?
                             investigation.investigationSubStatusByInvestigationSubStatus.displayName :
                             '';
+                        const subOccupation=  investigation?.investigatedPatientByInvestigatedPatientId?.subOccupationBySubOccupation?.displayName;
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:642]: same

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 9]
```diff
@@ -684,6 +639,9 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
...
+                        const parentOccupation=  investigation?.investigatedPatientByInvestigatedPatientId?.subOccupationBySubOccupation?.parentOccupation;
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:643]: same

**Review by ItayBenMoshe — CHANGES_REQUESTED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Review by MayaSasson — DISMISSED** [review]

**Review by OrelKaravani — APPROVED** [review]

**Review by roymaniv — APPROVED** [review]

**Pull request merged**